eclipse-theia/theia-blueprint
# json: fix json support

#### What it does


Fixes: #77

The following commit fixes JSON support for the application as the builtin extension was too recent and used unsupported APIs.

#### How to test


1. open a `settings.json` file
2. confirm that autocompletion works correctly and the schema is supported

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
